docs/additional-functionality/rapids-shuffle.md

```diff
@@ -282,30 +282,13 @@ In this section, we are using a docker container built using the sample dockerfi
 
 ### Spark App Configuration
 
-1. Choose the version of the shuffle manager that matches your Spark version.
-   Currently we support:
-
-    | Spark Shim    | spark.shuffle.manager value                              |
-    | --------------| -------------------------------------------------------- |
-    | 3.0.1         | com.nvidia.spark.rapids.spark301.RapidsShuffleManager    |
-    | 3.0.2         | com.nvidia.spark.rapids.spark302.RapidsShuffleManager    |
```

Comment:
How did we end up solving incompatible ShuffleManager base across Apache Spark versions. I thought having a single facade class across shims required dynamic bytecode generation  at Runtime depending on the shim because compile-time method signatures cannot be consistent for each Spark version?

Reply:
I tested everything after 3.1.1+ and it worked. I have reverted the change as it wasn't my original intention to remove the duplicate ShuffleManagers. The original intention was to get rid of the dynamic lookup of the shims at runtime.